src/main/java/frc/robot/RobotContainer.java

```diff
@@ -9,6 +9,11 @@
 import frc.robot.commands.ExampleCommand;
 import frc.robot.subsystems.DriveTrain;
 import frc.robot.subsystems.ExampleSubsystem;
+import frc.robot.subsystems.PhotonVision;
+
+import org.photonvision.PhotonVersion;
+
+import edu.wpi.first.wpilibj.XboxController;
```

Comment:
Rearrange the imports in a more fashionable manner, for example like a semi pyramid.

```suggestion
import org.photonvision.PhotonVersion;

import frc.robot.subsystems.PhotonVision;
import edu.wpi.first.wpilibj.XboxController;
```